packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.test.js

```diff
@@ -9,20 +9,44 @@ import path from 'path'
 import OverridesResolverPlugin from './overrides-plugin'
 
 const PROJECT_DIR = `src/configs/webpack/test`
+const EXTENDS_TARGET = '@salesforce/retail-react-app'
+
+// Our main webpack config will add the leading '/' if not present in package.json
+// so we can expect the '/' to be present here
+const OVERRIDES_DIR = '/overrides'
+
+// Files in this map are the files we expect to see in overrides,
+// reflecting the files found in src/configs/webpack/test/overrides
 const FS_READ_HASHMAP = new Map([
     ['exists', ['src/configs/webpack/test/overrides/exists.jsx', ['.', 'jsx']]],
+    ['newExtension', ['src/configs/webpack/test/overrides/newExtension.tsx', ['.', 'tsx']]],
     ['path/data', ['src/configs/webpack/test/overrides/path/data.js', ['.', 'js']]],
     ['path', ['/index.jsx', ['index', '.', 'jsx']]],
+    ['path/index.mock', ['/index.mock.jsx', ['.', 'jsx']]],
     ['path/nested/icon', ['src/configs/webpack/test/overrides/path/nested/icon.svg', ['.', 'svg']]]
 ])
-const EXTENDS_TARGET = 'retail-react-app'
-const REWRITE_DIR = 'src/configs/webpack/test/overrides'
+const REWRITE_DIR = PROJECT_DIR + OVERRIDES_DIR
 const options = {
-    overridesDir: '/overrides',
-    extends: ['retail-react-app'],
+    overridesDir: OVERRIDES_DIR,
+    extends: [EXTENDS_TARGET],
     projectDir: PROJECT_DIR
 }
 
+// Helper function. Expects an object with 2 properties: path and request
+const createRequestContextWith = (req) => {
+    return {
+        _ResolverCachePluginCacheMiss: true,
+        context: {
+            // We don't modify or read the issuer in overrides so we can
+            // leave this as a constant here
+            issuer: path.join('./', 'fake-file.js')
```

Comment:
```suggestion
            issuer: 'fake-file.js'
```
That's all this `path.join` does 🤷

Reply:
Applied. Lots of path.joins with './' have been addressed